### PR TITLE
Describe the use of unencrypted tokens in Nuki

### PR DESCRIPTION
**Pull request by pree**
## Proposed change

Describe the configuration option `Use a secure connection to the Bridge`.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104007
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nuki.markdown [hunk 1]
```diff
@@ -35,6 +35,8 @@ For faster updates, the callback function of the Nuki bridge can be used. This r
     description: Port of the Nuki bridge HTTP API, default is `8080`.
   Token:
     description: Token to authenticate against the Nuki bridge HTTP API.
+  Use a secure connection to the Bridge:
+    description: Use encrypted token for API calls to the bridge. This should only be deactivated, if you experience issues with the API (authentication, etc). Default is `True`.
```
Comment (FletcherAU) [inline source/_integrations/nuki.markdown]:
```suggestion
    description: Use encrypted token for API calls to the bridge. This should only be deactivated if you experience issues with the API (authentication, etc). Default is `True`.
```

Comment (pree) [reply source/_integrations/nuki.markdown]: Thanks!

File: source/_integrations/nuki.markdown [hunk 2]
```diff
@@ -35,6 +35,8 @@ For faster updates, the callback function of the Nuki bridge can be used. This r
     description: Port of the Nuki bridge HTTP API, default is `8080`.
   Token:
     description: Token to authenticate against the Nuki bridge HTTP API.
+  Use an encrypted token for authentication:
+    description: Use encrypted token for API calls to the bridge. This should only be deactivated if you experience issues with the API (authentication, etc). Default is `True`.
```
Comment (frenck) [inline source/_integrations/nuki.markdown]:
```suggestion
    description: Use an encrypted token for API calls to the bridge. This should only be deactivated if you experience issues with the API (authentication, etc). The default is `True`.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**